src/content/en/fundamentals/accessibility/semantics-aria/implicit-semantics.md

```diff
@@ -0,0 +1,40 @@
+project_path: /web/_project.yaml
```

Comment:
Any chance this file can be rolled into something else within semantics-aria? it's pretty short.